### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by gudarzzi** [projectdiscovery/asnmap#439] [General]
[FTL] Get "https://asn.projectdiscovery.io/api/v1/asnmap?ip=216.239.38.120&arch=amd64&go_version=go1.21.11&machine_id=1a84114327de713e51783a9c5c4cea6e532093212d8bbfac921b7c35d3eca998&os=ubuntu&utm_source=unknown&v=v1.1.1": dial tcp 104.196.155.173:443: connect: connection refused

whats wrong?

**Comment by GeorginaReeder** [comment]
Hi @gudarzzi ! Before we take a closer look at this, we'll need a little more information. What input were you using that resulted in this error? If you're also able to provide screenshots, that would be really helpful.

Tagging @dwisiswant0 in the meantime!
